### PR TITLE
Porting design kw

### PR DESCRIPTION
**Pull request by oysteoh**
Depends on #20

### CODE REVIEW

File: tests/test_formatting.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            os.path.join(root, "semeio"),
+            os.path.join(root, "setup.py"),
+            "--exclude",
+            "semeio/version.py",
+        ],
+    )
+
+    assert (
+        resp.exit_code == 0
+    ), "Black would still reformat one or ore files:\n{}".format(resp.output)
```
Comment (sondreso) [inline tests/test_formatting.py]: _one or **m**ore_?

File: semeio/jobs/design2params/design2params.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
+import logging
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def _create_handler(formatter, stream, level):
```
Comment (sondreso) [inline semeio/jobs/design2params/design2params.py]: Ref: https://github.com/equinor/semeio/pull/15#discussion_r362841311

File: semeio/jobs/design_kw/design_kw.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import sys
+import string
+import os
+import re
+import json
+import logging
+
+
+def _create_handler(formatter, stream, level):
```
Comment (sondreso) [inline semeio/jobs/design_kw/design_kw.py]: Ref: https://github.com/equinor/semeio/pull/15#discussion_r362841311

File: tests/jobs/design_kw/test_design_kw.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+def test_run_unmatched(tmpdir, input_data):
+    template_filename = "template_unmatched.yml.tmpl"
+    result_filename = "template.yml"
+    design_kw.run(template_filename, result_filename, "parameters.txt")
+
+    assert not os.path.isfile(design_kw._STATUS_FILE_NAME)
+
+
+def test_run_duplicate_keys(tmpdir, input_data):
```
Comment (sondreso) [inline tests/jobs/design_kw/test_design_kw.py]: tempdir variable is not used in any of the tests

**Comment by oysteoh** [comment]
@sondreso slightly modified after comments - how is the structure now? better? opinions?

**Pull request merged**